lib/stdlib/src/supervisor.erl

```diff
@@ -1481,9 +1494,9 @@ restart_multiple_children(Child, Children, SupName) ->
 restarting(Pid) when is_pid(Pid) -> ?restarting(Pid);
 restarting(RPid) -> RPid.
 
--spec try_again_restart(child_id() | {'restarting',pid()}) -> 'ok'.
-try_again_restart(TryAgainId) ->
-    gen_server:cast(self(), {try_again_restart, TryAgainId}).
+-spec try_again_restart(child_id() | {'restarting',pid()}, reference()) -> 'ok'.
+try_again_restart(TryAgainId, HRef) ->
+    gen_server:cast(self(), {try_again_restart, HRef, TryAgainId}).
```

Comment:
The hibernate after timeout message uses the format `{Tag, hibernate}`, so this should probably be `{Tag, try_again_restart, TryAgainId}` for consistency?

Reply:
Actually, I'd prefer to keep the atom first, for two reasons none of them I'm entirely sure about:
1. if this gets printed out to some debug or crash logs, and truncated, the atom is the readable key I don't want to get lost
2. I believe the BEAM has special instructions for matching an atom as the first element of a tuple, or at least that was the case at some point.